src/RamCondition.h

```diff
@@ -128,17 +128,18 @@ class RamConjunction : public RamCondition {
     }
 
 protected:
-    /** Left-hand side of conjunction */
-    std::unique_ptr<RamCondition> lhs;
-
-    /** Right-hand side of conjunction */
-    std::unique_ptr<RamCondition> rhs;
-
     bool equal(const RamNode& node) const override {
         assert(nullptr != dynamic_cast<const RamConjunction*>(&node));
         const auto& other = static_cast<const RamConjunction&>(node);
         return getLHS() == other.getLHS() && getRHS() == other.getRHS();
     }
+
+protected:
```

Comment:
The repeated `protected:` is not necessary. It's also repeated in many of the other Ram classes in this PR.

Reply:
I don't like that member variables are interleaved with methods. Either we have them first or last. I like the additional ```protected``` to show the boundary. However, I am easy - we should just experiment a little bit and make it consistent long-term.

Reply:
Our ordering standard is public, protected, private - so that the most generally useful information comes first. Within each of those, constants come first, then other member variables, then methods. (Brie/BTree are the exception and have many interleaved public/protected/private sections.)

We usually have public methods and private/protected member variables so it usually *looks* like methods, then variables. `equals()` is one of the few exceptions. Perhaps we could change to methods before variables in each public/protected/private section?